### DISCUSSION TITLE
Explore: UI improvements for log row context

### DISCUSSION THREAD

**Discussion by aitchjoe** [grafana/grafana#55667] [Observability]
When click "Show context" button on log browser, the popup window is small and little user friendly, so what we want:

* Large as the main window size.
* Fixed.
* Just highlight the original line, no need split on the middle.

We have compared the Kibana UI, when "Show context" they just goto new window instead of popup.

**Comment by svennergr** [comment]
Thanks for your feedback @aitchjoe.
We currently do some improvements around the "show context" feature, thus this feedback is very valuable!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
